src/frontend/sharedComponents/CameraView.tsx

```diff
@@ -122,7 +110,7 @@ export const CameraView = ({onAddPress}: Props) => {
         }
 
         onAddPress({
-          capturePromise: rotatePhoto(pic, accelerometerMeasurement.current),
+          capturePromise: Promise.resolve({uri: pic.uri}),
```

Comment:
somewhat lazy but updating the thing that uses `capturePromise` can become a potentially larger change in how the draft observation stuff works. preferred to leave that as is for the scope of this PR.